data/tutorials/getting-started/3_01_ocaml_on_windows.md

```diff
@@ -92,6 +286,8 @@ easier way to get a working Windows environment on your machine.
 
 ### Visual Studio Code on Windows
 
+**If you use opam installation**, you will need to add opam switch prefix on your path that runs VSCode.
```

Comment:
Perhaps there could be a link here on how to add switch prefix to the path?

Reply:
Why is that needed at all? If it is needed, we should file a bug somewhere since that is quite unpleasant for a user.